src/app/frontend/common/components/endpoint/external/template.html

```diff
@@ -20,30 +20,32 @@
    Link is always shown independent of the endpoint capabilities to support HTTP.
 -->
 <div>
-  <div *ngFor="let endpoint of endpoints">
-    <div *ngFor="let port of endpoint?.ports">
-      <div>
-        <a href="http://{{endpoint.host}}:{{port.port}}"
-           target="_blank">
-          {{endpoint.host}}:{{port.port}}
-          <mat-icon class="kd-endpoint-icon">open_in_new</mat-icon>
-        </a>
+  <ng-container *ngIf="endpoints">
```

Comment:
Just for the record, all I added here was <ng-container *ngIf="endpoints"> and </ng-container>
everything else in this diff is caused by identation.

Reply:
Why is it needed actually? If array will be empty nothing will be changed.

Reply:
The page would not load without that. I assume it's some race condition.

Reply:
I'll take a look

Reply:
Fixed